probe/probe.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package probe
+
+
+import (
+	"fmt"
+	"syscall"
+    "strings"
+	"golang.org/x/mod/semver"
+
+)
+
+func PrintProbeResult() error{
+	AutoDetectKernelVersion, err := autoDetectKernelVersion()
```

Comment:
```suggestion
	hostKernelVersion, err := detectHostKernelVersion()
```

These names would make more sense imo. WDYT?